code/network/multi.cpp

```diff
@@ -932,11 +924,17 @@ void process_packet_normal(ubyte* data, header *header_info)
 			break; 
 
 		default:
-			nprintf(("Network", "Received packet with unknown type %d\n", data[0] ));
+			mprintf(("Received packet with unknown type %d\n", data[0] ));
 			header_info->bytes_processed = 10000;
 			break;
 
 	} // end switch
+
+	// this is for helping to diagnose misaligned packets.  The last sensible packet 
+	// is usually the culprit that needs to be analyzed.
+	if (Cmdline_dump_packet_type) {
+		mprintf(("Packet type of %d received.  Total amount of data processed from larger packet is %d.\n", data[0], header_info->bytes_processed));
```

Comment:
Then just remove the `Packet type of %d received.` part and this line will continue on the earlier one.